### DISCUSSION TITLE
How to cite ISCE?

### DISCUSSION THREAD

**Discussion by weiji14** [isce-framework/isce2#574] [Q&A] [answered]
Hi there,

Just wondering if there's a recommended way to cite ISCE? I did a quick search and found https://github.com/isce-framework/isce3/blob/0eb175f86f6c74d86c19ba9623f23054513fc657/doc/doxygen/index.dox#L61 that says `Please refer to the <a href="https://github-fn.jpl.nasa.gov/isce-3/isce/CITATION">CITATION</a> file.`, but that links seems to be internal to JPL only. If there's an official DOI or something for ISCE, that would be great for us to add into our reference list.

Thanks!

**Comment by EJFielding** [comment, marked as answer]
The citation that I use is this:
Rosen, P. A., E. Gurrola, G. F. Sacco, and H. Zebker (2012), The InSAR Scientific Computing Environment, paper presented at 9th European Conference on Synthetic Aperture Radar, Nuremberg, Germany, 23-26 April.

**Reply by weiji14** [reply]
Thanks @EJFielding! That [EUSAR 2012 conference paper](https://ieeexplore.ieee.org/abstract/document/6217174) looks good, didn't realize ISCE went so far back!

Oh, and I did some citation hoping and found a newer one (with a DOI) that might be suited for ISCE3, just posting in case people stumble on this too:
- Rosen, P. A., Gurrola, E. M., Agram, P., Cohen, J., Lavalle, M., Riel, B. V., Fattahi, H., Aivazis, M. A. G., Simons, M., & Buckley, S. M. (2018). The InSAR Scientific Computing Environment 3.0: A Flexible Framework for NISAR Operational and User-Led Science Processing. IGARSS 2018 - 2018 IEEE International Geoscience and Remote Sensing Symposium, 4897–4900. https://doi.org/10.1109/IGARSS.2018.8517504
